### DISCUSSION TITLE
Daily 11/10

### DISCUSSION THREAD

**Discussion by NataliaAlvarezIspc** [grupo-ispc-2023-2-0/workflix-app#183] [Daily]
Con respecto al proyecto:

- ¿Qué hice ayer?
- ¿Qué hice o haré hoy?
- Impedimentos o dificultades

**Comment by NataliaAlvarezIspc** [comment]
Ayer me instalé el InelliJ, bajé todos los cambios del repositorio y probé que todo me funcione correctamente. Funciónó el back pero me tiró un errror el android studio que no pude ver de que se trataba. Por la tarde participe de la clase sincrónica del instituto.
Hoy voy a probar nuevamente de emular la aplicación para tratar de corregir el error que me tira. También voy a hacer la tarea de inglés, ya que tengo clase a la noche con evaluación. Voy a participar en la clase de proyecto integrador a la tarde.
Dificultades: pocos recursos en la computadora que hacen que todo se enlentezca y me lleve mucho tiempo realizar cualquier acción.

**Comment by MarielaGomez** [comment]
Ayer estuve en clases, también adelantando cuestionarios que vengo un poco atrasada, haciendo proyectos paralelos para volcar lo que voy aprendiendo, lo que a la vez se dificulta afianzar los conocimientos del backend ya que es poco el tiempo que se tiene entre entrega y disponibilidad de práctica.

**Comment by erick-lpz** [comment]
Ayer no puede asistir a la clase y tampoco puede finalizar la funcionalidad de la activity de ver usuarios y la agregarle la parte del backend
Hoy voy terminar todo eso y después ver de como agrego las siguientes activity al dashboard admin
Impedimentos: Tuve que ser hospitalizado y hoy en la mañana tenía turno para un control ya que se me dificulta para caminar

**Comment by BraianTroncoso** [comment]
Ayer asistí a clases y estuve leyendo y viendo vídeos sobre la implementacion de lógin con varias librerias de spring como también la posibilidad de implementar JWT. También hoy asisití a clases, trato de llevar al dia muchas otras cuestiones pero se dificulta la falta de tiempo al realizar muchas tareas paralelamente.
Sin embargo veo que obtengo avances, android es prácticamente nuevo lo que veo pero me gusta.

**Comment by Valentino399** [comment]
Antes de ayer estuve enfermo y no pude lograr hacer nada y ayer me conecte a la meet y hoy también ahorita voy a ver videos sobre el curso de java que estaba haciendo para aprender mas del lenguaje
